fivetran/resource_connector.go

```diff
@@ -1822,6 +1827,10 @@ func resourceConnectorReadConfig(resp *fivetran.ConnectorCustomMergedDetailsResp
 		mapAddStr(c, "connection_method", v)
 	}
 
+	if v, ok := resp.Data.CustomConfig["is_single_table_mode"].(string); ok {
+		mapAddStr(c, "is_single_table_mode", v)
+	}
```

Comment:
```suggestion
	if v, ok := resp.Data.CustomConfig["is_single_table_mode"].(bool); ok {
		mapAddStr(c, "is_single_table_mode", boolToStr(v))
	}
```

API will always return boolean value for this field, so we have to transform it into a string:
https://github.com/fivetran/engineering/blob/main/integrations/file/src/com/fivetran/integrations/file/FileOAuthCredentials.java#L42

I also found that we have errors in our public docs:
https://fivetran.com/docs/rest-api/connectors/config#mergemoderequest
https://fivetran.com/docs/rest-api/connectors/config#mergemoderequest_1
https://fivetran.com/docs/rest-api/connectors/config#mergemoderequest_2
https://fivetran.com/docs/rest-api/connectors/config#mergemoderequest_3

Request will work, object mapper will do transformation automatically, but in response the boolean value will be returned, and request/response will be inconsistent.